exchanges.yml

```
@@ -261,24 +261,15 @@ components:
       type: object
       description: Object containing information about the exchange to be created.
       properties:
-        exchangeInformation:
+        expires:
+          type: string
+          description: The date and time (as specified by ISO 8601) that the exchange expires.
+        variables:
           type: object
-          properties:
-            ttl:
-              type: string
-              description: Time to live for the exchange (ms).
-            variables:
-              type: array
-              description: Template variables to be used in the exchange.
-              items:
-                type: object
-                properties:
-                  type:
-                    type: string
-                    description: The type of template.
-                  template:
-                    type: string
-                    description: The template itself.
+          description: Template variables to be used in the exchange.
+        openId:
+          type: object
+          description: Optional parameters to enable OID4* protocol for delivery.
```

```suggestion
          description: Optional parameters to enable OID4 protocol for delivery.
```

Is the official name for the "family" of protocols "OID4"?

---

Group didn't have strong feelings about this change, leaving it as is.